### DISCUSSION TITLE
Portable distribution (without installer)

### DISCUSSION THREAD

**Discussion by josephernest** [fontforge/fontforge#4876] [Q&A] [answered]
Is there a portable Windows distribution, i.e. a ZIP package containing all the files, including the .exe binaries, but *without* any installer?

**Comment by ctrlcctrlv** [comment, marked as answer]
@josephernest Yes, but you have to go into the «[Actions](https://github.com/fontforge/fontforge/actions/runs/1601165744)» tab to get into it. Click on the commit, then scroll to «Artifacts», the one you want is almost certainly `MINGW64`, but there's also MINGW32 for really old 32-bit Windows.

By the way, and @jtanx can certainly answer better than me, but I have built on Windows before and contributed to `fontforgebuilds`, the installer is not dangerous…it doesn't edit registry without you telling it to, and you can use it just as an extractor then zip up its results just fine. I have done it.

**Reply by josephernest** [reply]
Thanks @ctrlcctrlv 
I see this: 
![image](https://user-images.githubusercontent.com/6168083/148071209-68833439-f421-4017-bafa-d5e2b120b2bf.png)

Where can we download the .EXE files before the creation of the installer? I can't find where in this UI I can access to those files, can you point me to the right place with a link? Thanks!

**Reply by ctrlcctrlv** [reply]
I can't give you a link directly that will work forever. They're always changing and old versions are pruned. It's better you learn the UI, people on Google will see this too.

Scroll down to bottom of «Actions» commit page, choose "MINGW64", allow browser to download `MINGW64.zip`:
![image](https://user-images.githubusercontent.com/838783/148153082-3af55083-f4fc-4135-8559-6d46c6c4810f.png)

The file contains `.7z` files inside a `.zip` which has the portable executable:
![image](https://user-images.githubusercontent.com/838783/148153038-af942c5f-d322-4c30-900f-aedc70beb753.png)

**Reply by josephernest** [reply]
Thanks! I clicked on the first MINGW64, and I didn't see the bottom of the pages with artifacts, my bad! Solved!

**Reply by ctrlcctrlv** [reply]
No problem :) I too wish that GitHub provided more useful links, like a link that would always download the most recent artifact with a certain name.
